instaclustr/resource_cluster.go

```diff
@@ -520,6 +520,11 @@ func resourceCluster() *schema.Resource {
 										Optional: true,
 										ForceNew: true,
 									},
+									"redis_password_auth": {
```

Comment:
Might be worth adding this parameter to the tested Redis config in valid_redis_cluster_create.tf , so that we have a test case covering the change?

Reply:
Reference to INS-14580